src/store/audio.ts

```diff
@@ -42,19 +43,18 @@ async function generateUniqueIdAndQuery(
   audioItem = JSON.parse(JSON.stringify(audioItem)) as AudioItem;
   const audioQuery = audioItem.query;
   if (audioQuery != undefined) {
-    if (audioItem.engineId == undefined)
-      throw new Error(`audioItem.engineId is undefined`);
     audioQuery.outputSamplingRate =
-      state.engineSettings[audioItem.engineId].outputSamplingRate;
+      state.engineSettings[audioItem.voice.engineId].outputSamplingRate;
     audioQuery.outputStereo = state.savingSetting.outputStereo;
   }
 
   const data = new TextEncoder().encode(
     JSON.stringify([
       audioItem.text,
       audioQuery,
-      audioItem.engineId,
-      audioItem.styleId,
+      audioItem.voice.engineId,
+      audioItem.voice.speakerId,
+      audioItem.voice.styleId,
```

Comment:
もしかしたらvoiceをstringfyするのもかっこいいかも？
```suggestion
      audioItem.voice,
```